src/Cache/RedisStore.php

```diff
@@ -45,7 +45,9 @@ public function get(string $key, bool $withExpired = false)
         }
 
         $contents = $this->redis->get($key);
-        $contents = json_decode($contents, true);
+        if (null !== $contents) {
+            $contents = json_decode($contents, true);
```

Comment:
I think we can use null-coalesce operator here right?

```suggestion
       $contents = json_decode($contents ?? "", true);
```

Reply:
My solution is a bit faster:
- https://3v4l.org/L7r31/perf
- https://3v4l.org/ZTHVa/perf

Tried it a few times, came up with the same results.

Let me know, I'm fine either way.

Reply:
I didn't realize there is a performance impact with `??`. I am fine with initial implementation. Thank you!

Reply:
Me neither :D